RxCocoa/Common/DelegateProxyType.swift

```diff
@@ -40,7 +40,7 @@ This is more or less how it works.
                   |                                                           
       +-----------v-------------------------------+                           
       |                                           |                           
-      | Delegate proxy : DelegateProxyType        +-----+---->  Observable<T1>
+      | Delegate proxy: DelegateProxyType        +-----+---->  Observable<T1>
```

Comment:
I think `+-----+---->  Observable<T1>` needs to be moved one space to the right.